Firmware/Marlin_main.cpp

```diff
@@ -5850,7 +5850,62 @@ SERIAL_PROTOCOLPGM("\n\n");
         }
         break;
     }
-    /*!
+
+  /*!
+
+  ### M75 - Start the print job timer <a href="https://reprap.org/wiki/G-code#M75:_Start_the_print_job_timer">M75: Start the print job timer</a>
+  */
+    case 75: //M75 Start/Resume the print job timer
+    {
+        if ((!isPrintPaused) || (pause_time = 0)) {
```

Comment:
I think `||` here should be `&&` otherwise `pause_time` is always 0

When I run M75 when resuming a paused print, then `pause_time` is never given a value.

Reply:
And ofcourse `pause_time = 0` should be `pause_time == 0` :)